broadinstitute/pearl
# AR-87 registration and screening workflows

this enables the UI for participants to be prescreened for a particular study. This flow works either from the hub, if the user is already signed in, or from the home page as an anonymous user.

The routing of the frontend has been rewired to use more decentralized routing -- this allows the registration components to be used both inside and outside of a study-specific workflow.

This updates/cleans up a bunch of populate files and images too. 

TO TEST:
[[ singed-in flow ]]
0. repopulate ourhealth and hearthive portals
1. go to http://sandbox.hearthive.localhost:3001
2. register and get to the hub
![image](https://user-images.githubusercontent.com/2800795/214753809-2cd27e2f-ee18-438f-bf56-21613ef83e17.png)
3. Click "join" on the cardiomyopathy study.
4. Confirm that you are taken to a prescreener.
5. Answer "yes"
6. Confirm you are taken to a page showing an enrollee shortcode

[[anonymous flow]]

1. go to http://sandbox.ourhealth.localhost:3001/
2. click "Join OurHealth"
3. confirm you are taken to an eligiblity survey
4. if you enter an age <18, confirm you are taken to an ineligible page
5. If you enter an age >18 confirm you are taken to registration
6. After registration, confirm you are taken to a page showing an enrollee shortcode
